xtask/commands/release-command.ts

```diff
@@ -228,24 +228,41 @@ export class ReleaseCommand extends Command {
     }
   }
 
-  private async gitPush(repo: Repository) {
+  private async gitPush(repo: Repository, targets: ReleaseTarget[]) {
     const remote = repo.getRemote('origin');
+    const refspecs = [
+      'refs/heads/main:refs/heads/main',
+      ...targets.map(x => x.package.nextVersionedGitTag.tagRef).map(tagRef => `${tagRef}:${tagRef}`),
+    ];
+    const logRefspecs = () => {
+      for (const refspec of refspecs) {
+        const [src, dest] = refspec.split(':');
+        console.log(colors.dim(`  - ${src} -> ${dest}`));
+      }
+    };
     if (this.dryRun) {
       console.log(`${colors.info('[root]')} will push to: ${remote.url()}`);
+      logRefspecs();
       return;
     }
     if (this.githubToken == null) {
       console.log(`${colors.warn('[root]')} no github token found. skip push.`);
       return;
     }
-    await remote.push(['refs/heads/main:refs/heads/main'], {
-      credential: {
-        type: 'Plain',
-        password: this.githubToken,
-      },
-      remoteOptions: ['--tags'],
-    });
+    await remote.push(
```

Comment:
example for this code will be like:
<img width="621" alt="screenshot" src="https://github.com/user-attachments/assets/34469fde-1a46-4d1b-8e2c-2f4c0b08632b" />